File: DayOne/Java/Level1/ArraySplitterTest.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertArrayEquals;
import static org.junit.jupiter.api.Assertions.assertNotNull;

public class ArraySplitterTest{

    @Test
    
    public void testThatGetSplitArraySplitsGivenArrayIntoA2DArrayOfEvenAndOddPosition(){

    int[] array = {45, 60, 3, 10, 9, 22};
    int[][] actual = ArraySplitter.getSplitArray(array);
    int[][] expected = {{45, 3, 9}, {60, 10, 22}};
    assertArrayEquals(actual, expected);
    }
    

}

File: DayOne/Python/Level1/array_splitter_test.py
from unittest import TestCase
from array_splitter import *

class ArraySplitter(TestCase):
    def test_that_get_split_array_splits_given_array_into_a_2D_array_of_even_and_odd_position(self):
        array =[45, 60, 3, 10, 9, 22]
        expected = get_split_array(array)
        actual = [[45, 3, 9], [60, 10, 22]]
        self.assertEqual(actual, expected)

